### PR TITLE
Add Connection using SSH

### PR DESCRIPTION
**Pull request by Henry6612700**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Add them to this list so others can find it too and start contributing, however,
 ## Submitting a pull request
 
 1. [Fork](https://github.com/OtacilioN/awesome-hacktoberfest-2018/fork) and clone the repository.
-1. Create a new branch: `git checkout -b my-branch-name`.
+2. Create a new branch: `git checkout -b my-branch-name`.
```
Comment (aslafy-z) [inline CONTRIBUTING.md]:
```suggestion
1. Create a new branch: `git checkout -b my-branch-name`.
```

File: README.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
 > Don't forget to [read our contributing guide](https://github.com/OtacilioN/awesome-hacktoberfest-2019/blob/master/CONTRIBUTING.md)
 
 ## Contents
+- [Connecting to Github Using SSH] (https://help.github.com/en/articles/connecting-to-github-with-ssh)
```
Comment (aslafy-z) [inline README.md]:
```suggestion
- [Connecting to Github Using SSH](https://help.github.com/en/articles/connecting-to-github-with-ssh)
```

File: README.md [hunk 3]
```diff
@@ -56,6 +56,7 @@
 - [HelloWorld-Hacktoberfest](https://github.com/ghost1372/HelloWorld-Hacktoberfest)
 - [inspirational-quotes](https://github.com/vinitshahdeo/inspirational-quotes)
 - [HBD](https://github.com/vinitshahdeo/HBD)
+- [Simple Contribute] (https://github.com/KongkowITMedan/SimpleContribute)
```
[the remark this answers is not in the document]
Comment (Henry6612700) [reply README.md]: So, you ask me to delete the repo that I pull?

[the remark this answers is not in the document]
Comment (OtacilioN) [reply README.md]: You can accept the suggestion by clicking on commit suggestion.

**Comment by Henry6612700** [comment]
Ok. Thank you. I have been already update to your repository

**Pull request closed without merging**